blocks/carousel/carousel.css

```diff
@@ -165,7 +167,18 @@
     }
 
     .carousel .carousel-slide figure {
-        flex-flow: row nowrap;
-        gap: var(--spacer-layout-02);
```

Comment:
Same instruction as before... Instead of deleting/adding lines under `.carousel .carousel-slide figure`, please add a new grouping for your new code under `.carousel.testimonial .carousel-slide figure`.